core/src/network/reducers/get_entry.rs

```diff
@@ -190,7 +190,7 @@ mod tests {
         };
         let dht_data = DhtData {
             msg_id: String::from(""),
-            dna_hash: String::from(""),
+            dna_address: "".into(),
```

Comment:
Please use `String::new()` if you want to create an empty string

Reply:
@sphinxc0re yes and a `HashString` is not a `String`

Reply:
I think it might be better to implement `Default` for `HashString` and use `Default::default()` instead of the current instantiation

Reply:
mmm  that is an interesting idea

Reply:
I think you can just derive it, since this will pick the default value for the internal string